packages/plugin-browserify/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.19.1]
+### Added
+- Generate source maps from modified code ([#615](https://github.com/MetaMask/snaps-skunkworks/pull/615))
```

Comment:
Is this correct? Or should it be `Changed` and `Fixed 0.19.0 not being able to run`?

Reply:
Correct, the plugins run correctly.

It's the cli package itself that was broken, not the plugins.